Assets/Scripts/Models/Area/World.cs

```diff
@@ -478,6 +484,7 @@ private void TickEveryFrame(float deltaTime)
     {
         CharacterManager.Update(deltaTime);
         FurnitureManager.TickEveryFrame(deltaTime);
+        GameEventManager.Update(deltaTime);
         ShipManager.Update(deltaTime);
```

Comment:
can we somehow start unifying the Update methods? it's getting messy a bit, also it would be good if it can start with UpdateXXX.. UpdateEveryFrame.. UpdateFixedFrequency, .. ?